src/pages/admin/order/AdminOrderTableHistory.tsx

```diff
@@ -33,18 +33,23 @@ const EmptyLabel = styled.div`
 
 function AdminOrderTableHistory() {
   const { workspaceId, tableNumber } = useParams<{ workspaceId: string; tableNumber: string }>();
+  const [tableOrders, setTableOrders] = useState<PaginationResponse<Order>>(defaultPaginationValue);
   const { replaceLastPath } = useCustomNavigate();
-  const tableOrders = useRecoilValue(tableOrderPaginationResponseAtom);
   const { fetchWorkspaceTable } = useAdminOrder(workspaceId);
   const [selectedOrderId, setSelectedOrderId] = useState<number | null>(null);
 
-  const isEmptyWorkspaces = tableOrders.empty;
+  const isEmptyWorkspaces = tableOrders?.empty ?? true;
   const emptyMessage = '찾고자 하는 주문이 존재하지 않습니다.';
 
   const pageSize = 6;
 
+  const fetchAndSetWorkspaceTable = async (tableNum: number, page: number, size: number) => {
```

Comment:
```suggestion
  const fetchAndSetWorkspaceTable = async (tableNo: number, page: number, size: number) => {
```

줄인다면 no가 더 좋아보이기도 하네요~
취향으로 반영하시면 됩니다,